app/models.py

```diff
@@ -10,7 +10,7 @@
 )
 _name_field = Field(min_length=3, max_length=255)
 
-_field_ddos = Field(max_length=255)
+_field_max_for_input = Field(max_length=255) #Max lenght for avoid ddos ​​attacks
```

Comment:
Que tal `_length_constraint`? `field_max_for_input` é meio estranho de se ler

Além disso tem 3 erros no comentário:
1. Deveria ser `length`
2. `to avoid` ao invés de `for avoid`
3. É melhor escrever `DDoS` já que é uma sigla